week1/homework/src/server.js

```diff
@@ -10,11 +10,34 @@ function createServer(port) {
 
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    function sendResponse(statusNum, content) {
+      response.statusCode = statusNum;
+      response.writeHeader(200, { 'content-type': 'application/json' });
```

Comment:
You are now writing `statusNum` to `response.statusCode` and (regardless of the `statusNum` parameter) setting the header to `200` in `response.writeHeader()`.
You are mixing up two methods for setting headers: 
> `response.setHeader(name, value)`
> https://nodejs.org/api/http.html#http_response_setheader_name_value 

and

> `response.writeHead(statusCode[, statusMessage][, headers])`
> https://nodejs.org/api/http.html#http_response_writehead_statuscode_statusmessage_headers

Since you are already using `response.statusCode()`, `response.setHeader('content-type', 'application/json')` should be enough. 
```suggestion
      response.setHeader('content-type', 'application/json');
```

Reply:
thank you for the feedback 
appreciated try to do what you asked me